stko/optimizers/collapser.py

```diff
@@ -228,60 +231,571 @@ def optimize(self, mol):
             shutil.rmtree(output_dir)
         os.mkdir(output_dir)
 
-        BB_ids = list(set([
+        bb_ids = list(set([
             i.get_building_block_id() for i in mol.get_atom_infos()
         ]))
-        BB_atom_ids = {i: [] for i in BB_ids}
+        bb_atom_ids = {i: [] for i in bb_ids}
         for i in mol.get_atom_infos():
-            BB_atom_ids[i.get_building_block_id()].append(
+            bb_atom_ids[i.get_building_block_id()].append(
                 i.get_atom().get_id()
             )
 
-        # Translate each BB along BB_COM_vectors `step`.
-        # `step` is the proportion of the BB_COM_vectors that is moved.
+        # Translate each bb along bb_COM_vectors `step`.
+        # `step` is the proportion of the bb_COM_vectors that is moved.
         step_no = 0
         step = self._step_size
         while not self._has_short_contacts(mol):
             # Update each step the building block vectors and distance.
```

Comment:
Doesn't make sense. At each step update the building block vectors and distance?

Reply:
step in Collapser means something different to step in CollapserMC

Reply:
OK the setence didn't make much sense originally but now I think I understand.